src/main/java/org/junit/rules/Timeout.java

```diff
@@ -143,16 +143,17 @@ protected final boolean getLookingForStuckThread() {
      * @since 4.12
      */
     protected Statement createFailOnTimeoutStatement(
-            Statement statement) throws Exception {
+            Statement statement, Description description) throws Exception {
```

Comment:
Unfortunately this method is protected. Changing the signature may break someone.